sesam.py

```diff
@@ -2212,8 +2257,46 @@ def format(self, record):
     parser.add_argument('-force', dest='force', required=False, action='store_true',
                         help="force the command to run (only for 'upload' and 'download' commands) for non-dev "
                              "subscriptions")
+
+    parser.add_argument("--system-placeholder", metavar="<string>",
```

Comment:
For every one of these new arguments, I would add a note that they are relevant only when working on connectors to their `help` string.

Reply:
done